### PR TITLE
Added alt attribute to the "concept" diagram on the Streams API page.

### PR DESCRIPTION
**Pull request by goshdarnheck**
#### Summary
As mentioned in https://github.com/mdn/content/issues/19334 there are many missing or incomplete alt attributes for images. This PR adds some descriptive alt text for 8 images on the CSS filter property page.

#### Motivation
Accessibility! For anyone using a screen reader, or if the images don't load etc.

#### Supporting details
List of missing alt attributes: https://docs.google.com/spreadsheets/d/1cDHee6mwDMlY6zANEmg8MjR8tL-wrCS86uawpWZ4XbA/edit?usp=sharing

#### Related issues
https://github.com/mdn/content/issues/19334

#### Metadata
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/streams_api/index.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ But this has never been available to JavaScript before. Previously, if we wanted
 
 With Streams being available to JavaScript, this all changes — you can now start processing raw data with JavaScript bit by bit as soon as it is available on the client-side, without needing to generate a buffer, string, or blob.
 
-![](concept.png)
+![A diagram illustrating the basic concept of the stream API. First, there is a cloud labelled "Fetch from network", beside it are arrows labelled "data" which point to some gears labelled "Process Data". From there, more arrows labelled "data" point towards an illustration of a page with images and text labelled "Render Data".](concept.png)
```
Comment (estelle) [inline files/en-us/web/api/streams_api/index.md:30]: when writing alternative image text, you want to include what information the image conveys, If it's a photon in a bio, then a description makes sense. In this case, the image is explaining the streaming process, so maybe:

`The basic concept of the stream API is data is fetched from the network in several data packets. The data is processed, and then sent to the browser in a stream of data packets.`

**Review by estelle — COMMENTED** [review]
Alt text should convey the information the image conveys. Please see my comment in line.

thanks

File: files/en-us/web/api/streams_api/index.md (hunk 1 above)
Comment (goshdarnheck) [reply files/en-us/web/api/streams_api/index.md:30]: That's so much better! I will push an update soon.
Thanks for the help, I will be able to write much better alt text moving forward 😁

**Comment by estelle** [comment]
uh oh. Looks like a Git bork happened. Good luck.

**Comment by goshdarnheck** [comment]
yep, I made a little rebase mistake, I will create a new PR

**Pull request closed without merging**